modules/schema_validation.lua

```diff
@@ -169,6 +189,7 @@ local function table_contains(table, element)
 end
 
 
+-- @function Validator:CompareType
```

Comment:
@LuxoftAKutsan,
Due to meta-table usage we require to *manually* shows that it's a method of the final `Validator` table.